polaris-react/src/components/Filters/components/FilterPill/FilterPill.scss

```
@@ -0,0 +1,110 @@
+@import '../../../../styles/common';
+
+$border-width: 1px;
+$border-radius: 32px;
+
+.FilterButton {
+  background: var(--p-surface-subdued);
+  border-radius: $border-radius;
+  border: var(--p-border-subdued) dashed $border-width;
+  cursor: pointer;
+  color: var(--p-text);
+
+  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
+  @include focus-ring($border-width: $border-width);
+  // stylelint-disable-next-line selector-max-specificity -- generated by polaris-migrator DO NOT COPY
+  &.focusedFilterButton:focus-visible:not(:active) {
+    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
+    @include focus-ring($style: focused);
+  }
+
+  &:hover,
+  &:focus,
+  &:active {
+    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
+    @include recolor-icon(var(--p-icon-hovered));
+    background: var(--p-surface-hovered);
+    border-color: var(--p-border);
+  }
+
+  &.disabledFilterButton {
+    background: var(--p-surface-disabled);
+    border-color: var(--p-border-disabled);
+    color: var(--p-text-disabled);
+    cursor: default;
+  }
+
+  &.ActiveFilterButton {
+    background: var(--p-surface);
+    border-style: solid;
+  }
+
+  &::after {
+    border-radius: $border-radius;
+  }
+}
+
+.PlainButton {
+  background: none;
+  color: inherit;
+  border: none;
+  padding: 0;
+  font: inherit;
+  cursor: inherit;
+  outline: inherit;
+  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
+  @include recolor-icon(var(--p-icon));
+
+  &[aria-disabled='true'] {
+    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
+    @include recolor-icon(var(--p-icon-disabled));
+  }
+}
+
+.ToggleButton {
+  padding: calc(var(--p-space-1) - var(--p-space-025))
+    calc(var(--p-space-2) - var(--p-space-025))
+    calc(var(--p-space-1) - var(--p-space-025))
+    calc(var(--p-space-3) - var(--p-space-025));
```

```suggestion
  padding: 0.75rem 1.75rem 0.75rem 2.75rem;
```

Hard coding and disabling stylelint with a description is the best route for declaring values that require nuance. That allows us to easily recognize disable patterns that point to gaps in our tokens through the data we collect from stylelint-polaris/coverage errors.

Can you share context on what the 3/7/11 spacing accomplishes here that 
4/8/12 can't?

---

Decided to hardcode the height (and disable/comment with stylelint) here rather than use these padding values to achieve the desired height.